prover/protocol/dedicated/emulated/limbs.go

```diff
@@ -0,0 +1,121 @@
+package emulated
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/consensys/linea-monorepo/prover/maths/field"
+	"github.com/consensys/linea-monorepo/prover/protocol/wizard"
+	"github.com/consensys/linea-monorepo/prover/zkevm/prover/common"
+)
+
+func limbsToBigInt(res *big.Int, buf []*big.Int, limbs Limbs, loc int, nbBitsPerLimb int, run *wizard.ProverRuntime) error {
```

Comment:
I think it will become gradually crucial that we have good abstraction for limbs. (Thinking generally). I can see when debugging the glue that it is already an issue. I'll work on a proposal tomorrow but I think that'd be good if we could incorporate that in the POC